.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Pull Request
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+
+jobs:
```

Comment:
Please use SwiftLint to check the code.
https://github.com/marketplace/actions/github-action-for-swiftlint

If there is a better one, please let me know.

Reply:
Added Swiftlint https://github.com/ca-dp/bucketeer-ios-sdk/pull/96/commits/b0a5a389fe3fabbab93e519c67e0ecab1a821489

Reply:
Are those warnings will always show in the comments?
https://github.com/ca-dp/bucketeer-ios-sdk/actions/runs/4563585188/jobs/8052260131?pr=96

If so, I prefer to use `// swiftlint:disable...` on those lines.